proxies/shared/partials/Partial.Flows.CreateMessageBatchEndpoint.xml

```diff
@@ -5,7 +5,7 @@
             <Name>JavaScript.MessageBatches.Create.Validate</Name>
         </Step>
         <Step>
-            <Name>RaiseFault.400BadRequest</Name>
+            <Name>RaiseFault.4xxGeneric</Name>
```

Comment:
Is it worth swapping out `RaiseFault.400BadRequest` for `RaiseFault.4xxGeneric` in Partial.Flows.CreateMessageEndpoint.xml?
It would keep things consistent and `RaiseFault.400BadRequest` could be removed.

Reply:
`RaiseFault.400BadRequest` has now been deprecated